src/gleam/base.gleam

```diff
@@ -11,15 +11,13 @@ pub fn encode64(input: BitString, padding: Bool) -> String {
   }
 }
 
-if erlang {
-  external fn do_encode64(BitString) -> String =
-    "base64" "encode"
-}
```

Comment:
Could you revert the syntax changes please? 🙏

Reply:
This isn't a requirement anymore is it @lpil @thelinuxlich as https://github.com/gleam-lang/stdlib/commit/43ae440ad66571cc41fef8e20dd852b170a3dc2b got merged?